### PR TITLE
#35 Removed validation for requiring keyword_matches flag to be true

### PR DESCRIPTION
**Pull request by KidA78**
- Removed validation
- Updated unit test.

### CODE REVIEW

File: test/Posts/PostsTest.php [hunk 1]
```diff
@@ -43,11 +43,12 @@ public function testLookup() {
     * @expectedException Traackr\MissingParameterException
```
Comment (dchancogne) [inline test/Posts/PostsTest.php]: Since not exception is being raised, this directive probably needs to be removed.

**Comment by sprak3000** [comment]
👍

File: test/Posts/PostsTest.php [hunk 2]
```diff
@@ -43,11 +43,12 @@ public function testLookup() {
...
     */
    public function testSearchInvalidParameter() {
```
Comment (sprak3000) [inline test/Posts/PostsTest.php]: Should update the name of the test here. The parameter is no longer invalid. Are we just testing a successful search with no posts?

**Comment by sprak3000** [comment]
👍

**Pull request merged**